tests/ttnn/unit_tests/operations/test_all_gather_matmul.py

```diff
@@ -311,7 +335,7 @@ def test_all_gather_matmul_on_t3000_post_commit(
     )
 
 
-# @skip_for_wormhole_b0() # Used to disable test
+# @skip_for_wormhole_b0()  # Used to disable test
```

Comment:
Delete?

Reply:
Same thing, used for debug/perf measurement purposes when you only want to run specific tests